Update windows.md (#22733)
DataDog/documentation: content/en/agent/basic_agent_usage/windows.md

   <div class="alert alert-info">If you need to install a specific version of the Agent, see the <a href="https://ddagent-windows-stable.s3.amazonaws.com/installers_v2.json">installer list</a>.</div>

> 2. Run the installer by opening `datadog-agent-7-latest.amd64.msi` (you will be asked to provide your Administrator credentials).

Reviewer: ```suggestion
2. Run the installer by opening `datadog-agent-7-latest.amd64.msi`. When prompted, enter your Administrator credentials.
```

To install the Agent with the command line:

1. Download the [Datadog Agent installer][1].

> 2. Open Command or Powershell prompt as (**Administrator**

Reviewer: Is it missing a `)` or more content?
